apps/storybook/src/CheckboxGroup.stories.tsx

```diff
@@ -18,20 +18,23 @@ export const Normal = () => (
       { value: 'LCY', label: 'City' },
       { value: 'SEN', label: 'Southend' },
     ]}
-    values={optionsKnob(
-      'Values',
-      {
-        Heathrow: 'LHR',
-        Gatwick: 'LGW',
-        Stansted: 'STN',
-        Luton: 'LTN',
-        City: 'LCY',
-        Southend: 'SEN',
-      },
-      ['LHR'],
-      {
-        display: 'inline-check',
-      },
-    )}
+    values={
+      optionsKnob(
+        'Values',
+        {
+          Heathrow: 'LHR',
+          Gatwick: 'LGW',
+          Stansted: 'STN',
+          Luton: 'LTN',
+          City: 'LCY',
+          Southend: 'SEN',
+        },
+        ['LHR'],
+        {
+          display: 'inline-check',
+        },
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      ) as any
```

Comment:
optionsKnob returns a const type when it should be an array of const. Taking suggestions or ill take another look at the end of the ticket